Products/Zuul/catalog/model_catalog_init.py

```diff
@@ -311,12 +342,13 @@ def init_model_catalog(collection_name=ZENOSS_MODEL_COLLECTION_NAME):
     index_client.init(config)
     return index_client
 
-def run(processor_count=8, hard=False, root="", indexes=None, types=[]):
+
+def run(processor_count=8, hard=False, root="", indexes=None, types=[], terminate=None, toggle_debug=None):
```

Comment:
Technically `types` shouldn't be a literal mutable there, but this is only called once so it's academic.